pnpm-workspace.yaml

```diff
@@ -0,0 +1,3 @@
+packages:
+  - addon
+  - test-app
```

Comment:
would it cause any issues when workspace is missing?

Reply:
nope